tests/test_util.py

```diff
@@ -6,7 +6,7 @@
 import torch
 from pyro import util
 
-pytestmark = pytest.mark.stage('test_util')
+pytestmark = pytest.mark.stage('unit')
```

Comment:
These tests were not running at all, since we don't have a `test_util` stage in `.travis.yml`.